Makefile

```diff
@@ -65,38 +65,46 @@ crossbuild-package-stage:
 	mv build/mackerel-agent-stage build-linux-386/
 
 rpm: crossbuild-package
+	mkdir -p rpmbuild
 	MACKEREL_AGENT_NAME=$(MACKEREL_AGENT_NAME) _tools/packaging/prepare-rpm-build.sh
-	rpmbuild --define "_sourcedir `pwd`/packaging/rpm-build/src" --define "_builddir `pwd`/build-linux-386" \
-	      --define "_version ${CURRENT_VERSION}" --define "buildarch noarch" \
-				-bb packaging/rpm-build/$(MACKEREL_AGENT_NAME).spec
+	docker run --rm -v "$(PWD)":/workspace -v "$(PWD)/rpmbuild":/rpmbuild astj/mackerel-rpm-builder:c6 \
```

Comment:
`--rm` is not for images but for containers.  We cannot reuse this container in L75 by removing `--rm`. (It just causes stopped container remaining)